lib/backend/model/block.go

```diff
@@ -92,6 +92,10 @@ func (options BlockListOptions) KeyFromDefaultPath(path string) Key {
 		return nil
 	}
 	cidrStr := strings.Replace(r[0][1], "-", "/", 1)
+	if _, _, err := net.ParseCIDR(cidrStr); err != nil {
```

Comment:
Looks like we're calling `net.ParseCIDR` twice in this function now - I think we can just call it once, and handle the error below where we're currently ignoring it.

Reply:
great , I did not notice that